app/experimenter/experiments/constants/nimbus.py

```diff
@@ -123,6 +123,11 @@ class Channel(models.TextChoices):
         ],
     }
 
+    class DocumentationLink(models.TextChoices):
+        DS_JIRA = "DS_JIRA", "Data Science Jira Ticket"
+        DESIGN_DOC = "DESIGN_DOC", "Experiment Design Document"
+        ENG_TICKET = "ENG_TICKET", "Engineering Ticket (Bugzilla/Jira/Github)"
```

Comment:
nit: GitHub

Reply:
I wonder if we should add another enum for "PI Ticket (Jira)", but that's probably unrelated to this PR, so I can file a new issue shortly.

Reply:
Good point, I was using [this list](https://github.com/mozilla/experimenter/issues/4400#issuecomment-758216153) from @k88hudson so maybe we can come up with a more official list?

Reply:
Filed as #4446

Reply:
TIL you can file from a comment!